core/templates/pages/exploration-editor-page/exploration-editor-page.component.spec.ts

```diff
@@ -505,21 +508,27 @@ describe('Exploration editor page component', function() {
     });
 
     it('should show the user help modal for editor tutorial', () => {
-      spyOn($uibModal, 'open').and.returnValue({
-        result: $q.resolve('editor')
-      });
+      spyOn(ngbModal, 'open').and.returnValue(
+        {
+          componentInstance: new MockNgbModalRef(),
+          result: $q.resolve('editor')
+        } as NgbModalRef
+      );
       ctrl.showUserHelpModal();
       $rootScope.$apply();
-      expect($uibModal.open).toHaveBeenCalled();
+      expect(ngbModal.open).toHaveBeenCalled();
```

Comment:
Please split the tests into setup, pre-check, action and post-check by adding a new line between them

Reply:
Please do this for all the tests

Reply:
Done